components/AvalancheForecastZoneMap.tsx

```diff
@@ -75,7 +76,9 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
         {isReady && <AvalancheCenterForecastZonePolygons key={center} center_id={center} setRegion={setRegion} date={date} />}
       </MapView>
       <SafeAreaView>
-        <DangerScale px="4" width="100%" position="absolute" top="12" />
+        <View flex={1}>
+          <DangerScale px={4} width="100%" position="absolute" top={12} />
+        </View>
```

Comment:
not sure why this worked before, but in this PR the absolute positioning stopped respecting the safe area - it needs to be absolute positioning relative to a View that's sized to the safe area.

it's possible that #66 is caused by a similar issue, not sure yet!